tests/Microsoft.Bot.Streaming.Tests/StreamingRequestHandlerTests.cs

```diff
@@ -79,7 +79,7 @@ public void CanBeConstructedWithAWebSocket()
             // Arrange 
 
             // Act
-            var handler = new StreamingRequestHandler(new MockBot(), new BotFrameworkHttpAdapter(), new FauxSock());
+            var handler = new StreamingRequestHandler(new MockBot(), new BotFrameworkHttpAdapter(), new FauxSock(), logger: null);
```

Comment:
This makes me nervous, is my new StreamingRequestHandler for WebSockets ambiguous? Could this cause build issues for developers who have sub-classed StreamingRequestHandler?

Reply:
Nevermind, the WebSocket+Audience constructor has Audience as an optional parameter whereas the new NamedPipe+Audience constructor requires Audience.

I'll update the WebSocket+Audience constructor to require Audience as is expected.